Detections/ASimWebSession/SuspiciousUACryptoMiners.yaml

```diff
@@ -1,24 +1,30 @@
 id: 8cbc3215-fa58-4bd6-aaaa-f0029c351730
 name: A host is potentially running a crypto miner (ASIM Web Session schema)
 description: |
-  'This rule identifies a web request with a user agent header known to belong to a crypto miner. This indicates a crypto miner may have infected the client machine.<br><br>This rule uses the Advanced SIEM Information Model (ASIM) and supports any web session source that complies with ASIM. To use this Analytics Rule, deploy the [Advanced SIEM information Model (ASIM)](https://aka.ms/DeployASIM).'
+    'This rule identifies a web request with a user agent header known to belong to a crypto miner. This indicates a crypto miner may have infected the client machine.<br><br>This rule uses the Advanced SIEM Information Model (ASIM) and supports any web session source that complies with ASIM. To use this Analytics Rule, deploy the [Advanced SIEM information Model (ASIM)](https://aka.ms/DeployASIM).'
```

Comment:
Append to the description:

"<br><br>You can add custom crypto mining indicating User-Agent headers using a watchlist, for more information refer to the [UnusualUserAgents Watchlist](aka.ms/ASimUnusualUserAgentsWatchlist)"

and a version of this to the other detections.